src/wallet/db.h

```diff
@@ -86,7 +87,7 @@ class CDBEnv
 };
 
 /** Get CDBEnv and database filename given a wallet path. */
-CDBEnv* GetWalletEnv(const fs::path& wallet_path, std::string& database_filename);
+std::shared_ptr<CDBEnv> GetWalletEnv(const fs::path& wallet_path, std::string& database_filename);
```

Comment:
typedef `std::shared_ptr<CDBEnv>` something? CDBEnvRef for instance.

Reply:
> typedef std::shared_ptr<CDBEnv> something? CDBEnvRef for instance

I don't think this is a good idea. I'd be reluctant to introduce yet another wallet database type, especially one that obfuscates what it represents and how it relates to CDBEnv. Use of ref type aliases has been justified in other places on the basis that where types are widely used, having aliases makes it easier to switch implementations without having to update a lot of existing code. But that justification doesn't apply here because CDBEnv is internal to db.h/db.cpp and not widely used.

Reply:
:+1: